pkg/router/router_test.go

```diff
@@ -94,7 +100,7 @@ func TestMain(m *testing.M) {
 		DefaultCertificateDir: workdir,
 		ReloadFn:              func(shutdown bool) error { return nil },
 		TemplatePath:          "../../images/router/haproxy/conf/haproxy-config.template",
-		ReloadInterval:        15 * time.Second,
+		ReloadInterval:        reloadInterval,
```

Comment:
Why such a big jump? And why do we then `* 2` a little later on?

Reply:
I had to make sure the router is reloaded, the way I'm doing it is by sleeping more than the `reloadInterval`.
I didn't want the unit test to last >15 seconds, so I dropped the reloadInterval to a more appropriate value.